cni/network/network.go

```diff
@@ -384,7 +385,11 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 			res.Print()
 		}
 
-		log.Printf("[cni-net] ADD command completed for pod %v with IPs:%+v err:%v.", k8sPodName, ipamAddResult.ipv4Result.IPs, err)
+		if ipamAddResult.ipv4Result.IPs != nil {
```

Comment:
We need to check here, otherwise without line 387 log, if ipv4Result object is empty, then try to get IPs: ipamAddResult.ipv4Result.IPs, the program will get crashed and exited

Reply:
isn't that behaviour same even before? we didn't see any crash before..